kola-denylist.yaml

```diff
@@ -55,9 +55,5 @@
 - pattern: ext.config.shared.networking.default-network-behavior-change
   tracker: ''
 
-# Temporary to unblock COSA CI
-- pattern: ext.config.shared.clhm.network-device-info
-  tracker: https://github.com/openshift/os/issues/1041
```

Comment:
We would need to keep this entry and add something like:
```
  osversion:
    - rhel-8.6
```
to disable it on RHCOS 8 only

Reply:
done